### PR TITLE
fix(package): add a pre-publish compile step

### PR DESCRIPTION
**Pull request by kumarharsh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,9 +8,7 @@
     "postcss-plugin",
     "css-modules"
   ],
-  "files": [
```
Comment (princed) [inline package.json:11]: npm will pack everything without `files` property

File: package.json [hunk 2]
```diff
@@ -33,6 +32,8 @@
     "eslint-plugin-import": "^2.2.0"
   },
   "scripts": {
+    "build": "babel --out-dir lib index.js",
+    "prepublish": "npm run build",
     "test": "eslint . && ava"
```
Comment (princed) [inline package.json]: Let's run `eslint *.js` then

**Review by princed — CHANGES_REQUESTED** [review]
Thanks for the PR!
I'd suggest some tweaks.

File: package.json (hunk 1 above)
Comment (kumarharsh) [reply package.json:11]: so, it should be `files: ['index.js', 'lib']`, right?

File: package.json (hunk 2 above)
Comment (kumarharsh) [reply package.json]: sure

File: package.json (hunk 1 above)
Comment (princed) [reply package.json:11]: Looks like index.js is a bit redundant, but let's see it as ability to run not compiled code.

**Review by princed — APPROVED** [review]

**Comment by princed** [comment]
Published as 1.1.0

**Pull request merged**